src/coreclr/inc/sarray.h

```diff
@@ -70,15 +70,7 @@ class SArray
     void Append(ELEMENT elem)
     {
         WRAPPER_NO_CONTRACT;
-        *Append() = elem;
-    }
-
-    ELEMENT AppendEx(ELEMENT elem)
-    {
-        WRAPPER_NO_CONTRACT;
-
-        *Append() = elem;
-        return elem;
```

Comment:
Does it matter that `AppendEx` no longer returns the `elem` appended to any callers?

Reply:
Nope. I removed the API because it wasn't used.